index.js

```diff
@@ -16,7 +16,6 @@ module.exports = {
   extends: [
     'airbnb',
     'prettier',
-    'prettier/react',
```

Comment:
As per `8.0.0` all plugins are bundled into the main `prettier`: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21